perception/autoware_map_based_prediction/src/map_based_prediction_node.cpp

```diff
@@ -1114,7 +1115,7 @@ void MapBasedPredictionNode::objectsCallback(const TrackedObjects::ConstSharedPt
 
         // If predicted reference path is empty, assume this object is out of the lane
         if (ref_paths.empty()) {
-          PredictedPath predicted_path = path_generator_->generatePathForLowSpeedVehicle(
```

Comment:
@YoshiRi @soblin 
Is `generatePathForLowSpeedVehicle` right for the empty ref_paths object?
I guess it should be `generatePathForOffLaneVehicle`.